src/infra/AnalyticsReporter.ts

```diff
@@ -40,10 +44,10 @@ export class AnalyticsReporter implements AnalyticsService {
 
   /**
    * Formats the event data for the api which includes adapting verticalKey to verticalConfigId
```

Comment:
nit: missing period at the end here and in a few other files' docs (e.g. AnalyticsEventType, AnalyticsResponse, HttpRequesterService)

Reply:
The missing periods should be addressed in #20